app/assets/javascripts/admin/addon/templates/components/admin-watched-word.hbs

```diff
@@ -1,4 +1,4 @@
-{{d-icon "times"}} {{word.word}}
+<span role="button" onclick={{action "deleteRecord"}} class="delete-word-record">{{d-icon "times"}}</span> {{word.word}}
```

Comment:
Is there a reason why you preferred `span role="button"` instead of just `button`? I'm not sure if there is any semantic or styling difference.

Reply:
Yeah, the `button` element breaks the styling.